test/WebSites/RazorWebSite/Controllers/PartialViewEngineController.cs

```diff
@@ -42,5 +42,15 @@ public IActionResult PartialWithModel()
             };
             return PartialView(model);
         }
+
+        public IActionResult RenderPartialWithModel()
```

Comment:
I think it's good to have a functional test specifically because we block with GetAwaiter().GetResult(), and I believe that that has the potential to be host dependent.